tasks/clone.js

```diff
@@ -6,15 +6,15 @@ module.exports = function(grunt) {
   var templateHelper = require('./lib/templateHelper').init(grunt);
   var _ = require('underscore');
 
-  function cloneAppTemplates(filepath, cloneDepth) {
+  function cloneAppTemplates(filepath, cloneDepth, useSsh) {
     grunt.config.set('gitclone', {});
 
     var globalJson = grunt.file.readJSON(filepath);
 
     templateHelper.walkObjects(globalJson.show, function (obj) {
       var prefix = 'gitclone.' + obj.id + '.options';
       if (_.has(obj, 'repoUrl') && _.has(obj, 'repoBranch')) {
-        var repoUrl = obj.repoUrl.replace('git://github.com/','git@github.com:');
+        var repoUrl = useSsh ? obj.repoUrl.replace('git://github.com/','git@github.com:') : obj.repoUrl;
```

Comment:
It was changed to use SSH so that we could do other things with the repos including pushing changes. We do this in the release pipeline. In this scenario, we do not, and I don't want to have to wrap the grunt command in the Jenkensifile in an sshagent block, so I made it configurable.

```
grunt clean copy clone --use-ssh true
```